gcovr/templates/base.html

```diff
@@ -16,21 +16,29 @@
   </head>
 
   <body>
-    <h1>{{info.head}}</h1>
+    <header>
+      <h1>{{info.head}}</h1>
 
-    <hr/>
+      <hr/>
 
-    <div class="summary">
-      {% block summary %}
-      {% endblock %}
-    </div>
+      <div class="summary">
+        {% block summary %}
+        {% endblock %}
+      </div>
 
-    <hr/>
+      <hr/>
+    </header>
 
-    {% block content %}
-    {% endblock %}
+    <navigation>
+      {% block navigation %}
+      {% endblock %}
+    </navigation>
 
-    <hr/>
+    <article>
+      {% block content %}
+      {% endblock %}
+      <hr/>
+    </article>
```

Comment:
I don't think using `<article>` makes sense here. If you want to surround this part with a HTML element, consider `<main>` or `<div>`.